pcsx2/DEV9/PacketReader/IP/UDP/DHCP/DHCP_Options.cpp

```diff
@@ -66,7 +66,7 @@ namespace PacketReader::IP::UDP::DHCP
 		NetLib::WriteByteArray(buffer, offset, dnsServers.size() * 4, (u8*)&dnsServers[0]);
 	}
 
-	DHCPopHostName::DHCPopHostName(std::string name)
+	DHCPopHostName::DHCPopHostName(std::string& name)
```

Comment:
Can these be made const? (both here and in DHCPopMSGStr & DHCPopClassID constructors)

Reply:
Uh, that's what I was actually going to do, slipped my mind.